{"name": "test_add[get_datas_add0]", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <pratice_demo.pratice_calculator_advanced.test_calc_new.TestCalculator object at 0x00000227C9B9A490>\nget_calc = <pratice_demo.pratice_calculator_advanced.demo.calc.Calculator object at 0x00000227C9B9A520>, get_datas_add = [999999999.9, 1, 10000000000.9]\n\n    @allure.story(\"计算器的加法测试\")\n    @pytest.mark.run(order=1)\n    def test_add(self, get_calc, get_datas_add):\n        # 假设最大能输入999999999\n        if get_datas_add[0] > 999999999 or get_datas_add[1] > 999999999:\n            print(\"超出最大数值999999999，不允许输出\")\n>           assert False\nE           assert False\n\ntest_calc_new.py:71: AssertionError"}, "attachments": [{"name": "stdout", "source": "a447e3a0-ec6e-4b75-ac24-d85c6c0379f0-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "get_datas_add", "value": "[999999999.9, 1, 10000000000.9]"}], "start": 1612345233052, "stop": 1612345233053, "uuid": "0ea3923c-fca9-4552-b0d1-548a26f3cc62", "historyId": "aecad3b7e8d3158f3cda373dbaac3ce1", "testCaseId": "3d63d6b6e535a2d30dc56b2e66da2d0a", "fullName": "test_calc_new.TestCalculator#test_add", "labels": [{"name": "feature", "value": "测试计算器"}, {"name": "story", "value": "计算器的加法测试"}, {"name": "tag", "value": "run(order=1)"}, {"name": "suite", "value": "test_calc_new"}, {"name": "subSuite", "value": "TestCalculator"}, {"name": "host", "value": "Naraku15"}, {"name": "thread", "value": "6112-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calc_new"}]}